src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -60,5 +76,27 @@ public function setMink(Mink $mink)
      */
     public function setMinkParameters(array $parameters)
     {
+        $this->minkParameters = $parameters;
+    }
+
+    private function prepareSessionIfNeeded()
```

Comment:
Nope i can create hook for this and create proper suite that will prepare cookie only when selenium is running.